index.html

```diff
@@ -47,7 +47,7 @@ <h3 class="publishedOn">{{publishedOn}}</h3>
   integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g="
   crossorigin="anonymous"></script>
 <script src="scripts/projectData.js"></script>
-<script src="scripts/index.js"></script>
+<script src="scripts/views/index.js"></script>
 <script src="scripts/pageView.js"></script>
```

Comment:
You also have a lot more js files that you now need to include in your index.html (e.g. controllers)